tests.py

```diff
@@ -216,6 +216,60 @@ def test_create_or_update_create_time_catch_request_error(self, m):
                                                                "time",
                                                                "times"))
 
+    def test_create_or_update_time_with_junk_string_duration(self):
```

Comment:
For tests that make an actual API call (`_create_or_update` makes an API call) you should mock the `requests` method that will be invoked, in this case `requests.post`